### PR TITLE
Added note about Startup Boost blocking auto-open-devtools-for-tabs

### PR DESCRIPTION
**Pull request by ryanborMSFT**
`auto-open-devtools-for-tabs` requires there to be *no* running Edge processes. Most users have the Startup Boost feature enabled, which keeps a minimal Edge process running in the background. This interferes with the auto-open-devtools-for-tabs feature and would need to be disabled to use it (or users would have to taskkill the Startup Boost Edge process before running the auto-open-devtools-for-tabs command).

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/overview.md [hunk 1]
```diff
@@ -129,6 +129,8 @@ To have DevTools automatically open whenever you open a new tab in the browser:
    microsoft-edge-dev --auto-open-devtools-for-tabs
    ```
    
+> [!IMPORTANT]
+> There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. More information about Startup Boost can be found here: [Get help with startup boose - Microsoft Support](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/overview.md:133]:
```suggestion
> There must be no running Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable Startup Boost in `edge://settings/system`. For more information, see [Get help with startup boost](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442

Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/overview.md:133]:
```suggestion

###### Disabling startup boost

> [!IMPORTANT]
> There must be no running Microsoft Edge processes when launching via `auto-open-devtools-for-tabs`. You may need to disable **Startup boost** in `edge://settings/system`. For more information, see [Get help with startup boost](https://support.microsoft.com/topic/get-help-with-startup-boost-ebef73ed-5c72-462f-8726-512782c5e442).
```
Rendered view (GitHub version):
https://github.com/ryanborMSFT/edge-developer/blob/patch-1/microsoft-edge/devtools-guide-chromium/overview.md#have-devtools-automatically-open-when-you-open-a-new-tab

Given the present style sheet, we're moving toward h2/h4/h6.  The containing heading is h3.  h6 should produce a suitable size at LMC (different than GitHub's rendering of heading sizes).

A heading is good to provide a skimmable label for an Alert eg an Important note.

File: microsoft-edge/devtools-guide-chromium/overview.md [hunk 2]
```diff
@@ -129,6 +129,8 @@ To have DevTools automatically open whenever you open a new tab in the browser:
...
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/overview.md:134]: Should some of this wording be added from PR main page, for more clarification?

auto-open-devtools-for-tabs requires there to be no running Edge processes. 

Most users have the Startup Boost feature enabled, which keeps a minimal Edge process running in the background. 

This interferes with the auto-open-devtools-for-tabs feature and would need to be disabled to use it.

Or users would have to task-kill the Startup Boost Edge process before running the auto-open-devtools-for-tabs command.

File: microsoft-edge/devtools-guide-chromium/overview.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/overview.md:133]: sent chat

**Review by mikehoffms — APPROVED** [review]

**Review by JasonAndrewWriter — APPROVED** [review]

**Pull request merged**